.github/workflows/cd-pipeline.yaml

```diff
@@ -112,50 +111,91 @@ jobs:
           VIB_ENV_S3_URL: s3://${{ secrets.AWS_S3_BUCKET }}/bitnami
           VIB_ENV_S3_USERNAME: ${{ secrets.AWS_ACCESS_KEY_ID }}
           VIB_ENV_S3_PASSWORD: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+  update-index:
+    runs-on: ubuntu-latest
+    needs:
+      - vib-publish
+    name: Update charts index
+    outputs:
+      result: ${{ steps.update-index.outputs.result }}
+    if: ${{ needs.vib-publish.result == 'success' }}
+    steps:
+      - uses: actions/download-artifact@v3
+        with:
+          path: ~/artifacts
       # If we perform a checkout of the master branch, we will find conflicts with the submodules
       - uses: actions/checkout@v2
         with:
           ref: 'index'
           path: index
-          token: ${{ secrets.BITNAMI_BOT_TOKEN }}
-      - name: Install and configure aws-cli, helm and yq
+      - name: Install and configure aws-cli, helm and jq
         run: |
           # AWS CLI
           sudo DEBIAN_FRONTEND=noninteractive apt-get install -y awscli
           aws configure set region us-east-1
           aws configure set aws_access_key_id ${{ secrets.AWS_ACCESS_KEY_ID }}
           aws configure set aws_secret_access_key ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws configure set source_profile default
-            # helm
+          # helm
           HELM_TARBALL="helm-v3.8.1-linux-amd64.tar.gz"
           curl -SsLfO "https://get.helm.sh/${HELM_TARBALL}" && sudo tar xf "$HELM_TARBALL" --strip-components 1 -C /usr/local/bin
-          # yq
-          sudo curl -SsLf https://github.com/mikefarah/yq/releases/download/v4.23.1/yq_linux_amd64 -o /usr/local/bin/yq && sudo chmod +x /usr/local/bin/yq
-      - name: Fetch chart and update index
+          # jq
+          sudo curl -SsLf https://github.com/stedolan/jq/releases/download/jq-1.5/jq-linux64 -o /usr/local/bin/jq && sudo chmod +x /usr/local/bin/jq
```

Comment:
AFAIK, you don't need to install `jq` here, as it already comes installed. You can check our [ci-pipeline](https://github.com/bitnami/charts/blob/f870f9cea589adec6387b1faf73066330a50eaf2/.github/workflows/ci-pipeline.yaml#L37) where we are using it with an out-of-the-box ubuntu-latest.

Reply:
Ohh, okay. Thanks for the heads-up 👍